content/agent/changelog.md

```diff
@@ -7,7 +7,43 @@ docs: "DOCS-1093"
 
 {{< note >}}You can find the full changelog, contributor list and assets for NGINX Agent in the [GitHub repository](https://github.com/nginx/agent/releases).{{< /note >}}
 
-See the list of supported Operating Systems and architectures in the [Technical Specifications]({{< relref "/agent/technical-specifications.md" >}}).
+See the list of supported Operating Systems and architectures in the [Technical Specifications]({{< relref "./technical-specifications.md" >}}).
```

Comment:
I thought we were converting all of these relrefs to "absolute" paths. Did I misunderstand?

Reply:
They should all be absolute paths: I've even taken to using `ref`over `relref` to reinforce this pattern.

@JTorreG This looks like an error.

Reply:
```suggestion
See the list of supported Operating Systems and architectures in the [Technical Specifications]({{< relref "/technical-specifications.md" >}}).
```